doc/MAINLINE_MODS.md

```diff
@@ -0,0 +1,57 @@
+# Getting Mods Into The Repository
+
+Cataclysm: Dark Days Ahead is not only moddable, but ships with a number of mods available for users to select from even without having to obtain mods from a third party source.
+
+## Why have mainlined mods?
+
+The benefits of having mods mainlined include:
+
+*  Visibility to a large number of users and potential contributors
+*  The ability to use the CleverRaven issue tracker to report problems, as well as manage pull requests for adding to or fixing mods in the repository
+*  Mods in mainline that demonstrate usage of a game feature which exists only for mods (i.e. not used by the core DDA game itself) help ensure that feature stays present in the game.
+
+There are however things that might be expected but are not guaranteed:
+
+*  Guarantee of maintenance of content -- core devs are mostly focused on the main game itself; although some might pick up mods as a side task, people who continue working on the core game tend to like the feel of the core game.
+*  Guarantee of maintenance of used features -- over the course of development it may become impossible to keep a feature, due to performance needs or maintenance burden.
+*  Guarantee of sole authorship -- once in the repository, the mod is considered community content. While the developers might defer to an active maintainer of a mod, if they become inactive anyone else can step forward.
+
+## What kind of mods can be in the repository as mainlined mods?
+
+There are three primary categories of mods:
+
+*  Content mods, which provide some kind of distinct experience from the core Dark Days Ahead game. This could be a change to lore, a change to a specific part of gameplay.
+*  User Experience (UX) mods, which alter the look and feel of the game interface itself.
+*  Development mods, which aren't "mods" in the typical sense but are instead there to ease the transition between "incomplete feature" and "incomplete feature", when a feature in the core game is sufficiently incomplete that the developers believe it needs to be optional to minimize disruption to players.
```

Comment:
between "incomplete feature" and "incomplete feature",

*between "incomplete feature" and "complete feature" ?